singlePost.php

```diff
@@ -0,0 +1,45 @@
+<?php
+require_once 'connectToDB.php';
+require_once 'src/Models/PostModel.php';
+
+$db = connectToDB();
+$postModel = new PostModel($db);
+
+session_start();
+
+$singlePostDetails = $postModel->getSinglePostById($_GET['id']);
```

Comment:
You should build in some check incase the post isn't found.

I would be happy if you just redirected back to index.php since the only way this will happen is if someone starts mucking around with the URL.